src/relentless/simulate/hoomd.py

```diff
@@ -1188,6 +1214,8 @@ def __init__(
             self.system = system
             self.rdf_params = rdf_params
             self.constraints = constraints if constraints is not None else {}
+            self.bonds = bonds
+            self.exclusion = exclusions
```

Comment:
This would be `exclusions` and come first, following the other code patterns
```suggestion
            self.exclusions = exclusions
            self.bonds = bonds
```